src/main/java/cs2103/v15_1j/jimjim/ui/JJUI.java

```diff
@@ -45,6 +47,11 @@ public void showTaskView() {
 	public void refreshUI(){
 		mainViewController.updateData(getDataLists());
 	}
+	
+	public void refreshUI(UIFeedback feedback){
+		mainViewController.updateData(getDataLists());
+		feedback.execute(mainViewController);
```

Comment:
I do call the old refreshUI method, like when the program first initiates and runs